libs/dh/globalization/feature-language-picker/src/lib/language-picker/dh-language-picker.component.ts

```diff
@@ -31,10 +33,18 @@ import { map, Observable } from 'rxjs';
 })
 export class DhLanguagePickerComponent {
   activeLanguage$: Observable<DisplayLanguage> =
-    this.transloco.langChanges$.pipe(map(toDisplayLanguage));
+    this.transloco.langChanges$.pipe(
+      map(toDisplayLanguage),
+      tap((language) => {
+        (this.dateAdapter as WattDateAdapter).setActiveLocale(language);
```

Comment:
suggestion: what about using a service to set the locale instead of exposing `DateAdapter` and `WattDateAdapter`?

We can provide the service as part of `WattDanishDatetimeModule.forRoot()`. Then we inject it here and in `WattDateAdapter` where we listen to language changes and set the locale. Wouldn't that work?

(So something similar to `TranslocoService`)

Reply:
How will you listen for language changes without making Watt depending on transloco? (this is the reason why I did it like this, and to keep it simple).

Reply:
I don't think Watt should depend on Transloco. My suggestion was to create a pattern similar to what Transloco is doing - to provide a simple service that is injected in Watt and here and use that to send values between app and Watt. So importing `DateAdapter` and `WattDateAdapter` wouldn't be necessary.